### ISSUE TITLE
Trying to use flutter version 2.2.1

### ISSUE THREAD

**Issue by josh-gree** [subosito/flutter-action#98]
Hi I would like to be able to use most recent version of flutter - with `subosito/flutter-action@v1` I get the following error;

```
unable to find release for 2.2.1'
```

In #85 it suggests to use `subosito/flutter-action@v1.5` but this gives the following error;

```
Unable to resolve action `subosito/flutter-action@v1.5`, unable to find version `v1.5`
```

Any help would be appreciated.

**Comment by ndahlquist** [comment]
I think the problem might be "v1.5.0" vs "v1.5". This is working for me: `subosito/flutter-action@v1.5.0`

**Comment by subosito** [comment]
We didn't release `v1.5`; we support major version like `v1` and more specific version like `v1.5.0`.

This issue is related to #99 issue. Please wait until the issue is resolved.

**Comment by subosito** [comment]
Just merged the #100 and released v1.5.1 as well as updated v1. Please retry.

**Comment by ndahlquist** [comment]
Confirmed fixed for me! Thank you! 🥇
